### PR TITLE
docs: update python 33 to python 35

### PR DESCRIPTION
**Pull request by bkmagnetron**
find_spec isn't available in Python 3.2.3 it was added in Python 3.4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,8 +105,8 @@ Templates give you full control of each component of your application.
 Sometimes your application is simple enough and you don't want to bother with templates. In that case, you can let OpenShift inspect your source code and create the required components automatically for you:
 
 ```bash
-$ oc new-app openshift/python-33-centos7~https://github.com/openshift/django-ex
-imageStreams/python-33-centos7
+$ oc new-app openshift/python-34-centos7~https://github.com/openshift/django-ex
+imageStreams/python-34-centos7
```
Comment (bparees) [inline README.md]: the correct 3.4 image name is centos/python-34-centos7, the openshift one is deprecated.

**Comment by PI-Victor** [comment]
@bkmagnetron can you please make the necessary adjustment to this PR? as suggested above?

**Comment by bparees** [comment]
please squash your commits and then lgtm.  though we also have python-35 now, so maybe you'd rather switch to that instead.

**Comment by PI-Victor** [comment]
@bkmagnetron ping! squash? :dancers: 
this is trivial and would be cool to merge fast

**Comment by bkmagnetron** [comment]
@PI-Victor done.

**Comment by bparees** [comment]
lgtm

**Pull request merged**